source/faq.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('_layouts.main')
+
+@section('body')
+  <section class="ud-hero" id="home">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="ud-faq">
+    <div class="shape">
+      <img src="{{ $page->baseUrl }}assets/images/faq/shape.svg" alt="shape" />
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <div class="ud-section-title text-center mx-auto">
+            <span>FAQ</span>
+            <h2>{{ $page->t("Any Questions? Answered")}}</h2>
```

Comment:
What is this?
I didn't understood the phrase.

Reply:
It is a default template phrase website.